common/src/main/java/com/microsoft/identity/common/crypto/AndroidAuthSdkStorageEncryptionManager.java

```diff
@@ -91,6 +91,6 @@ public AndroidAuthSdkStorageEncryptionManager(@NonNull final Context context,
         }
 
         Logger.warn(methodTag, "Cannot find a matching key to decrypt the given blob");
-        throw new IllegalStateException("Unknown keyIdentifier for cipher text: " + keyIdentifier);
+        return Collections.emptyList();
```

Comment:
So this is just reverting to what we have on dev, right?

Reply:
yes